### PR TITLE
Feature/#2796/ [ubs client cabinet] send notifications to user

### PR DESCRIPTION
**Pull request by Flash5482**
If user doesn't have any notification
135138217-ac1677ef-70c2-413c-8b8b-dba150c4f087
If user has notification
135138765-00a9bedf-3973-46dc-8ed4-72db7896ea1f

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -19,10 +19,7 @@
             "polyfills": "src/polyfills.ts",
             "tsConfig": "tsconfig.app.json",
             "aot": true,
-            "assets": [
```
Comment (NewArtMask) [inline angular.json:22]: Did you make some changes in angular.json or is it just the formatting? In the second case, please undo this and exclude angular.json from the pull request.

File: src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.scss [hunk 2]
```diff
@@ -77,3 +77,9 @@
 .arrow {
   width: 15px;
 }
+
+.count-unread-messages {
+  color: var(--ubs-primary-white);
+  margin-right: -25px;
```
Comment (nazarhlyvka) [inline src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.scss:83]: Use just margin

File: src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.ts [hunk 3]
```diff
@@ -18,7 +21,13 @@ export class UbsBaseSidebarComponent implements AfterViewInit {
   @ViewChild('drawer') drawer: MatDrawer;
   @ViewChild('sidebarContainer') sidebarContainer: ElementRef;
 
-  constructor(public breakpointObserver: BreakpointObserver) {}
+  constructor(public serviceUserMessages: UserMessagesService, public breakpointObserver: BreakpointObserver) {}
+
+  ngOnInit(): void {
+    this.serviceUserMessages
+      .getCountUnreadNotification()
+      .subscribe((response) => (this.serviceUserMessages.countOfNoReadeMessages = response));
```
Comment (nazarhlyvka) [inline src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.ts:29]: we have to add pipe and use takeUntil(this.destroy) or take(1) for unsubscribe
https://blog.bitsrc.io/6-ways-to-unsubscribe-from-observables-in-angular-ab912819a78f
or check our code

File: src/app/ubs-user/ubs-user-messages/notification-body/notification-body.component.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  templateUrl: './notification-body.component.html',
+  styleUrls: ['./notification-body.component.scss'],
+  encapsulation: ViewEncapsulation.ShadowDom
+})
+export class NotificationBodyComponent implements OnInit {
+  @Input() body: string;
+
+  constructor(public userMessagesService: UserMessagesService) {}
+
+  ngOnInit(): void {}
```
Comment (nazarhlyvka) [inline src/app/ubs-user/ubs-user-messages/notification-body/notification-body.component.ts:15]: remove hook and implements cuz it is empty

File: src/app/ubs-user/ubs-user-messages/ubs-user-messages.component.ts [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+    this.language = this.localStorage.getCurrentLanguage();
+    this.isLoadSpinner = true;
+    this.route.params.subscribe((val) => {
+      this.page = +this.route.snapshot.paramMap.get('pageId');
+      this.fetchNotification();
+    });
+  }
+
+  fetchNotification(): void {
+    this.userMessagesService.getNotification(this.language, this.page - 1, this.pageSize).subscribe(
```
Comment (nazarhlyvka) [inline src/app/ubs-user/ubs-user-messages/ubs-user-messages.component.ts:48]: add takeUntil() and unsubscribe in ngDestroy or take()

File: src/app/ubs-user/ubs-user-sidebar/ubs-user-sidebar.component.ts [hunk 6]
```diff
@@ -24,14 +25,14 @@ export class UbsUserSidebarComponent extends UbsBaseSidebarComponent implements
...
     }
   ];
 
-  constructor(public breakpointObserver: BreakpointObserver) {
-    super(breakpointObserver);
+  constructor(public service: UserMessagesService, public breakpointObserver: BreakpointObserver) {
+    super(service, breakpointObserver);
   }
 
   ngOnInit(): void {}
```
Comment (nazarhlyvka) [inline src/app/ubs-user/ubs-user-sidebar/ubs-user-sidebar.component.ts:38]: remove

File: src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.scss (hunk 2 above)
Comment (Flash5482) [reply src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.scss]: done

File: src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.ts (hunk 3 above)
Comment (Flash5482) [reply src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.ts:33]: Done

File: src/app/ubs-user/ubs-user-messages/notification-body/notification-body.component.ts (hunk 4 above)
Comment (Flash5482) [reply src/app/ubs-user/ubs-user-messages/notification-body/notification-body.component.ts]: Done

File: src/app/ubs-user/ubs-user-messages/ubs-user-messages.component.ts (hunk 5 above)
Comment (Flash5482) [reply src/app/ubs-user/ubs-user-messages/ubs-user-messages.component.ts]: Done

File: src/app/ubs-user/ubs-user-sidebar/ubs-user-sidebar.component.ts (hunk 6 above)
Comment (Flash5482) [reply src/app/ubs-user/ubs-user-sidebar/ubs-user-sidebar.component.ts]: Done

**Review by nazarhlyvka — APPROVED** [review]

**Review by KhrystynaPr — APPROVED** [review]

**Review by Bishko86 — APPROVED** [review]

**Review by NewArtMask — APPROVED** [review]

**Review by nazarf1 — APPROVED** [review]

**Pull request merged**